osu.Game/Screens/Select/Filter/SortMode.cs

```diff
@@ -29,6 +29,9 @@ public enum SortMode
         RankAchieved,
 
         [Description("Title")]
-        Title
+        Title,
+
+        [Description("Source")]
+        Source,
```

Comment:
Should have `Source` above `Title` to maintain the alphabetical enum order (as that applies to the control in game)